### ISSUE TITLE
Unable to open channel: unable to send funding request message: peer exiting

### ISSUE THREAD

**Issue by lloydrough** [LN-Zap/zap-desktop#3752]
After trying to open a channel to Zap with 220,000 sats. I keep getting this error. Different errors with other channels.
I can fund the wallet, just not open a channel
## Steps to Reproduce

- 1.
- 2.
- 3.
- 4.

## Your Environment

- Zap version:
- Operating System and version:

**Comment by bolatovumar** [comment]
@lloydrough you forgot to include the error you are getting.

**Comment by lloydrough** [comment]
The error is in the subject line

**Comment by lloydrough** [comment]
When running "node launcher' by Pierre Rochard..

Error information:

file could not be opened successfully
File "run.py", line 22, in File "node_launcher/gui/application.py", line
15, in __init__ File "node_launcher/node_set/node_set.py", line 41, in
__init__ File "node_launcher/node_set/lnd.py", line 90, in __init__ File
"node_launcher/services/lnd_software.py", line 15, in lnd File
"node_launcher/services/node_software.py", line 73, in executable_path File
"node_launcher/services/node_software.py", line 92, in extract File
"tarfile.py", line 1578, in open

**Comment by bolatovumar** [comment]
I haven't used the node launcher app myself and haven't seen this error. I searched around for this error message and couldn't find any mention of it anywhere in the LND source code. I wonder if it's coming from the node launcher itself?

**Comment by lloydrough** [comment]
The node launcher wont even launch without an error. Maybe it needs to be updated? How do I set up zap without this tool? I can see the channels but they wont connect.

**Comment by bolatovumar** [comment]
@lloydrough Zap can be used on its own without the Node Launcher.

**Comment by lloydrough** [comment]
Yes I can use it as a lightning wallet, but I want to use it as a lightning node. Doesn't that require some connection to my bitcoin node? The Pierre Rochard node launcher seems out of date. Where can I find the instructions to do this (run a lightning node) on a mac? I can see other lightning nodes on Zap but I cant successfully connect a channel.

**Comment by bolatovumar** [comment]
@lloydrough I never ran Zap in combination with an LND node on the same computer but BTCPay server allows Zap to connect to a standalone LN node. You can easily deploy BTCPay server as shown here (or alternatively there are guides for running it on your own hardware): https://docs.btcpayserver.org/LunaNodeWebDeployment/
